echopype/echodata/zarr_combine.py

```diff
@@ -1022,6 +1064,7 @@ def combine(
             chunks={},
             synchronizer=zarr.ThreadSynchronizer(),
             storage_options=storage_options,
+            backend_kwargs={"consolidated": consolidated},
```

Comment:
Why is this necessary?

Reply:
To be explicit that we are opening a consolidated zarr file.

Reply:
Where/how would we use/access  this information. I am not really familiar with `backend_kwards`.